content/200-concepts/100-components/02-prisma-client/053-client-extensions/120-query.mdx

```diff
@@ -35,7 +35,9 @@ Use the `query` extension component to modify queries. You can modify a custom q
 To create a custom query, use the following structure:
 
 ```ts
-const xprisma = prisma.$extends({
+const prisma = new PrismaClient()
+
+const extendedPrisma = prisma.$extends({
```

Comment:
Do we want to do this? After a brief glance I can't tell why we need both `prisma` and `extendedPrisma`
```suggestion
const prisma = new PrismaClient().$extends({
```

Reply:
In this example, the `query` extensions modify the methods available in Prisma Client. This might lead to undesired behavior if applied to the primary Prisma Client instance and one forgets about it. I'm not sure if we mention this anywhere but I think we generally recommend doing that on a separate Prisma Client instance

Reply:
I've made the update. Let me know if the wording could be better 😄